generator/src/test/java/com/linkedin/pegasus/generator/TestDataTemplateGeneratorCmdLineApp.java

```diff
@@ -178,11 +178,17 @@ private void testRunGenerator(String pegasusFilename, Map<String, String> expect
       Assert.assertTrue(generatedSource.contains("class " + pegasusTypeName),
           "Incorrect generated class name.");
 
-      // Match the generated annotation (use regex since this is error-prone)
+      // Validate the generated annotation (use a strange regex-based comparison since temp files can be weird e.g. /var vs. /private/var)
```

Comment:
Users have been building in Mac from long time ago.
why is this previously not caught?

Reply:
1. This test was added recently.
2. It's possible that only some Mac systems have this temp folder quirk, as it doesn't seem to happen for all Mac builds.